### DISCUSSION TITLE
Docker: Hosting more than one Supabase project per container?

### DISCUSSION THREAD

**Discussion by oguruma1218** [supabase/supabase#15317] [Questions]
I deployed Supabase in Docker on my local machine. In the Supabase Studio, I don't see an option to add a new project like I do with the Supabase hosted version...

Is it possible to host more than one project in a single Docker instance? Or do you have to create a new Docker instance for each project?

**Comment by GaryAustin1** [comment]
Just 1 project.

**Reply by GaryAustin1** [reply]
Related  https://github.com/orgs/supabase/discussions/5968

More discussions here if you search.
